### ISSUE TITLE
Innovation Unit making action-inject-ssm-secrets repo public

### ISSUE THREAD

**Issue by productmike** [department-of-veterans-affairs/va.gov-cms#13521]
Placeholder to unblock #12640 and #12641

**Comment by productmike** [comment]
DM slacks re: PO approvals of #12640 and #12641:

Dawn Pruitt
  3:02 PM
question regarding "done" and closing items in github:
tim has two tickets where closing is dependent upon a repo becoming public/dependency on another team.
Do we still count those as done for our team and keep open?
For reporting, right now I am counting them since tim did his work (and associated points).
I have a note by the burndown report for those two items and the dependency.
Wanted to see what you thought as PO Erika since I know the COR and Dave still look at them





3:02
if you don't want me to count those 10 points in the numbers at the top that's okay too


Erika Washburn
  3:03 PM
Those issues are a large dependency on an OCTO-DE team that I'm not even familiar with. I know he said it is Platform, but it isnt even them- it's the innovation unit. I would say those are done for our work. Maybe we need to open up another issue? But from my perspective we've done all we can do that's why I think open another issue for the other part


Dawn Pruitt
  3:04 PM
thanks! Yes you're right, it's not platform and tim did say some weird oit process LOL


Erika Washburn
  3:04 PM
yes. Innovation Unit is directly under Charles. idk what to do about it, but I'll try to get some advice from Mike C
:thank_you:
2



Erika Washburn
  3:20 PM
We all (Dawn I think you're on that thread)  just received a very positive and helpful update on Tim's issue.
:thank_you:
1



Mike Hollen (he/him/his)
  5:01 PM
not on that thread (or at least don't see it) but I open a very (very) generic placeholder so we can close these other two out
https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/gh/department-of-veterans-affairs/va.gov-cms/13521
